### ISSUE TITLE
IPR033162  tubulin specific chaperone cofactor D, Alp1

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#2068]
IPR033162

tubulin specific chaperone cofactor D, Alp1

maps to 
GTPase activator activity
is this the correct activity for this subunit?
seems odd?

**Comment by asangrador** [comment]
Not necessarily. In mammals cofactors C, D (Alp1) , and E assemble and act together as a beta-tubulin GTPase activating protein (GAP) (PMID:10831612). These factors also come together in the tubulin pathway in yeast, but I don't think the GAP activity has been shown for them, may be for cofactor C? (PMID:  23576550).

**Comment by ValWood** [comment]
This one is armadillo repeat so maybe it is just the scaffold? not the GTPase (if  this is not the case it would likely be a function of the complex rather than this particular subunit alone?)

**Comment by ValWood** [comment]
Actually SGD have cofactor C as the GAP, and I have copied this ...
https://www.yeastgenome.org/locus/S000006162#go
This is a useful ref for conservation of function to human:
PMID: 18376416

**Comment by asangrador** [comment]
Cofactor C has definitely GAP activity. Cofactors D and E seem to contribute to this activity in the complex and have some GAP activity PMID:10446175, 20740604 .  It is not very clear, but I think I will leave the annotation at the moment.

**Comment by ValWood** [comment]
but to make a "contributes_to" type annotation you need a "contributes_to" qualifier, and that can't be done with a mapping , so maybe here it would be better to exclude?

**Comment by asangrador** [comment]
UniPro has the 'GAP activity' annotation for the human protein (https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=Q9BTW9) . Also it seems that in vitro the cofactor C stimulates the GTPase activity of native tubulin, only with the cooperation of cofactor D [PMID:11847227], so it is necessary to stimulate the GTPase activy. We don't have qualifiers in InterPro, so if I remove the annotation the information will be lost. As we provide predicted IEA annotation we are a bit more relaxed in this.

**Comment by ValWood** [comment]
One problem is that the  GO assignment rule require the contributes_qualifier if the activity  is an emergent function of the complex.
We'll discuss these on the QC group and get back to you.

I agree, there are a  lot of supporting annotation, but 
this much more  recent paper dissecting the function of mammalian D doesn't refer to it as a "GTPase activator"
http://www.jbc.org/content/292/10/4336.full

Let's keep this one open for now while we do some back-tracking on this one. If I come across anyone I can ask in the community i'll follow up. If the original annotations are correct they need contributes_to, but I'm still unconvinced....I would need to read the papers in detail.

So leave the annotation for now until we get chance to dig around.

Val

@pgaudet

**Comment by asangrador** [comment]
That sounds like the best solution. Papers on the subject are quite confusing,  they don't seem to be decisive one way or another.

Cheers,
